src/docs/index.md

```diff
@@ -92,6 +92,7 @@ You might also find these docs useful:
 * [State management][]
 * [Animations][]
 
+The documentation follows the latest stable version of flutter
```

Comment:
Sorry to be so nit-picky, but please make the f in Flutter uppercase. Also, add a period at the end. Thx!